#!/usr/bin/env python
PACKAGE = "transbot_program"
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("lin_Kp", double_t, 0, "Kp in PID", 25, 0, 100)
gen.add("lin_Ki", double_t, 0, "Ki in PID", 0.0, 0, 100)
gen.add("lin_Kd", double_t, 0, "Kd in PID", 3, 0, 100)
gen.add("ang_Kp", double_t, 0, "Kp in PID", 25, 0, 100)
gen.add("ang_Ki", double_t, 0, "Ki in PID", 0, 0, 100)
gen.add("ang_Kd", double_t, 0, "Kd in PID", 3, 0, 100)
gen.add("linear", double_t, 0, "linear", 0.3, 0, 0.5)
gen.add("laserAngle", int_t, 0, "laserAngle", 35, 0, 180)
gen.add("ResponseDist", double_t, 0, "ResponseDist", 1.0, 0, 3)
exit(gen.generate(PACKAGE, "TransbotProgram", "TransbotProgram"))
